src/routes/api/completion/+server.ts

```diff
@@ -3,19 +3,19 @@ import { ChatbotCompletion } from '$lib';
 //TODO(max): Change this function to another protocol, nothing is being POSTed
 /** @type {import('./$types').RequestHandler} */
 export async function POST(params: { request: Request }) {
-	const { chatHistory } = await params.request.json();
+	const { chatHistory, input } = await params.request.json();
 
-	const completion_manager = new ChatbotCompletion(import.meta.env.VITE_OPENAI_API_KEY!, {
+	const completion_manager = new ChatbotCompletion(import.meta.env.VITE_OPENAI_API_KEY!, true, {
 		openai_model: import.meta.env.VITE_OPENAI_MODEL_NAME!
 	});
+	// ChatbotCompletion(openaikey, verbose, {model_name})
+	await completion_manager.setup();
 
-	const stream = await completion_manager.query(chatHistory);
+	const output = await completion_manager.query(chatHistory, input);
```

Comment:
We need to handle the error returned by query.

Reply:
I am working on making the error less common by making the llm listen to instructions.

Reply:
Even if the error is less common we still need to make sure that the error is handled properly

Reply:
I will just output the unparsed result